### PR TITLE
Fix seeding from backups

### PR DESCRIPTION
**Pull request by martinfurmanski**

### CODE REVIEW

**Review by phughk — APPROVED** [review]

**Comment by RagnarW** [comment]
@phughk Updated the tests with monitors and switched to comparing file keys.

File: enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/backup/ClusterSeedingIT.java [hunk 1]
```diff
@@ -183,4 +236,80 @@ public void shouldRestoreBySeedingSingleMember() throws Throwable
...
+
+    private void assertFileKeysAreTheSame( Map<File,Object> fileKeys ) throws IOException
+    {
+        for ( Map.Entry<File,Object> e : fileKeys.entrySet() )
+        {
+            File file = e.getKey();
+            Object oldKey = e.getValue();
+
+            BasicFileAttributes attr = Files.readAttributes( file.toPath(), BasicFileAttributes.class );
+            assertEquals( "Creation time for file: " + file, oldKey, attr.fileKey() );
```
Comment (martinfurmanski) [inline enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/backup/ClusterSeedingIT.java]: No longer creation time (text in assertion is wrong).

**Review by martinfurmanski — COMMENTED** [review]
Minor comment - otherwise all good.

**Comment by martinfurmanski** [comment]
I am not able to approve a PR in my own repo.

**Review by RagnarW — APPROVED** [review]

**Pull request merged**